features/consent/src/main/java/in/projecteka/jataayu/consent/extension/ConsentDataExtension.kt

```diff
@@ -0,0 +1,27 @@
+package `in`.projecteka.jataayu.consent.extension
+
+import `in`.projecteka.jataayu.core.model.Consent
+import `in`.projecteka.jataayu.core.model.RequestStatus.*
+
+enum class SortOrder { ASC, DESC }
+
+/* Sorts List of consents by last updated time */
+fun List<Consent>.getSortedConsentListByLastUpdatedDate(sortOrder: SortOrder = SortOrder.DESC): List<Consent> {
```

Comment:
can be 
`Sequence<Consent>.sortByLastUpdated(sortOrder: SortOrder): List<Consent> = 
when(sortOrder) {
        SortOrder.ASC -> {
            sortedBy { it.getLastUpdated() }
        }
        SortOrder.DESC -> {
            sortedByDescending { it.getLastUpdated() }
        }
}
`